File: 2003/IntroToVBNETPart2/Tutorial/LeapYear.vb
Imports System

Public Class LeapYear
	Shared Function isLeapYear ( ByVal year As Integer) As Boolean
		Dim leapYear As Boolean = False
		if year Mod 4 = 0  And year Mod 100 <> 0 Or year Mod 400 = 0 Then
			leapYear = True
		End If
		Return leapYear
	End Function

	Shared Sub Main ()
		Dim year As Integer
		For year = 1990 To 2010 
			Console.WriteLine ( "{0}   {1}", year, isLeapYear ( year ) )
		Next year
	End Sub
End Class

File: 2003/IntroToVBNETPart2/Tutorial/LeapYear.cs
using System ;

public class LeapYear
{
	static bool isLeapYear ( int year )
	{
		bool leapYear = false ;
		if ( year % 4 == 0 && year % 100 != 0 || year % 400 == 0 )
			leapYear = true ;
		return leapYear ;
	}

	public static void Main ()
	{
		for ( int year = 1990 ; year <= 2010 ; year++ )
			Console.WriteLine ( "{0}   {1}", year, isLeapYear ( year ) ) ;
	}
}